src/js/components/Button/stories/Basic.js

```diff
@@ -17,6 +17,15 @@ const BasicButtons = props => (
     <Box align="center" pad="medium">
       <Button primary label="Primary" onClick={() => {}} {...props} />
     </Box>
+    <Box align="center" pad="medium">
+      <Button
+        primary
+        label="Active Primary"
+        active
+        onClick={() => {}}
+        {...props}
+      />
+    </Box>
```

Comment:
Later on, we should restructure the stories so Active may have its own story for all types of buttons (primary, default...) same for disabled and hover. labeling for `interns`